packages/salesforcedx-utils-vscode/src/helpers/utils.ts

```diff
@@ -59,6 +60,21 @@ export const flushFilePath = (filePath: string): string => {
   return nativePath;
 };
 
+const isSymbolicLink = (path: string) => {
+  try {
+    let currentPath = resolve(path);
+    while(currentPath !== '/') {
+      const stats = lstatSync(currentPath);
+      const isLink = stats.isSymbolicLink();
+      if (isLink) return true;
+      currentPath = dirname(currentPath);
+    }
+    return false;
+  } catch (err) {
+      throw new Error('The path does not exist');
```

Comment:
Can we have more descriptive error message please? Also where would this error show up? output tab or dev tools?